test/verbose/verbose-flag.test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+// eslint-disable-next-line node/no-unpublished-require
+const { run } = require('../utils/test-utils');
+
+describe('verbose flag', () => {
+    it('should accept --verbose', () => {
```

Comment:
Need a better test here. i.e. we should test for lines in stdout that are found only in verbose stats.

Reply:
updated :+1:, Please take a look.